### PR TITLE
Added Strong password problem using python

### PR DESCRIPTION
**Pull request by Sulagna-Dutta-Roy**
Issue no: #10

### CODE REVIEW

**Comment by Kumar-laxmi** [comment]
@Sulagna-Dutta-Roy Add all 4 files in one PR

**Comment by Sulagna-Dutta-Roy** [comment]
Actually I wanted to make this pr only on python. But forget to mention in issue

**Comment by Kumar-laxmi** [comment]
@Sulagna-Dutta-Roy  Just give it a try in other languages too

**Comment by Sulagna-Dutta-Roy** [comment]
Sure 😄

**Comment by Kumar-laxmi** [comment]
@Sulagna-Dutta-Roy Any progress?

File: Python/Cryptography/Strong Password.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import sys
+
+
+def minimumNumber(n, password):
+    nums = "0123456789"
+    lower_case = "abcdefghijklmnopqrstuvwxyz"
+    upper = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+    special = "!@#$%^&*()-+"
+    counts = 4
+    if any(list(map(lambda x: x in nums,password))):
```
Comment (Kumar-laxmi) [inline Python/Cryptography/Strong Password.py:14]: Please add the purpose of each if-statement

Comment (Kumar-laxmi) [reply Python/Cryptography/Strong Password.py:14]: Explain each and every case OR edge condition that the if-statements are dealing

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**